### PR TITLE
Datawarehouse

### PR DESCRIPTION
**Pull request by schuyler1d**
This adds two features:
1. A frontend to the SQL interface to pull in campaign contacts from the data warehouse
2. A migrations regime that will have our dbs automatically run on startup (after pulling)

### CODE REVIEW

File: src/migrations/index.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
+import { r, Migrations } from '../server/models'
```
Comment (shakalee14) [inline src/migrations/index.js:1]: i like this better than having separate migration files.

File: src/server/index.js [hunk 2]
```diff
@@ -33,6 +34,9 @@ setupAuth0Passport()
 if (!process.env.SUPPRESS_SEED_CALLS) {
   seedZipCodes()
 }
+if (!process.env.SUPPRESS_MIGRATIONS) {
```
Comment (shakalee14) [inline src/server/index.js:37]: did we add this to the .env.template file?

**Review by shakalee14 — APPROVED** [review]

File: src/migrations/index.js (hunk 1 above)
Comment (sandramchung) [reply src/migrations/index.js:1]: ++

File: src/server/index.js (hunk 2 above)
Comment (schuyler1d) [reply src/server/index.js:37]: no because I don't think it should be the default.  We could add it to docs somewhere, I suppose

**Pull request merged**